spring-boot-project/spring-boot-actuator-autoconfigure/src/test/java/org/springframework/boot/actuate/autoconfigure/metrics/export/prometheus/PrometheusMetricsExportAutoConfigurationTests.java

```diff
@@ -64,6 +66,12 @@ void backsOffWithoutAClock() {
 		this.contextRunner.run((context) -> assertThat(context).doesNotHaveBean(PrometheusMeterRegistry.class));
 	}
 
+	@Test
+	void backsOfWhenExemplarSamplerIsPresent() {
+		this.contextRunner.withUserConfiguration(ExemplarsConfiguration.class)
+				.run((context) -> assertThat(context).doesNotHaveBean(DefaultExemplarSampler.class));
```

Comment:
asserting on the absence of `DefaultExemplarSampler` since a bean of type `ExemplarSampler` is present from `ExemplarsConfiguration`